microshift_storage/understanding-persistent-storage-microshift.adoc

```diff
@@ -23,6 +23,13 @@ include::modules/storage-persistent-storage-reclaim.adoc[leveloffset=+2]
 
 include::modules/storage-persistent-storage-pv.adoc[leveloffset=+1]
 
+ifdef::microshift[]
+[id="additional-resources"]
```

Comment:
For additional resource sections after a module, I don't think this anchor is necessary. I also tried to use it in a URL and it brought me to the top of the page anyways. Odds are you could probably delete this line.